src/tabs/failsafe.html

```diff
@@ -139,6 +139,12 @@
                                     </label>
                                     <div class="helpicon cf_tip" i18n_title="failsafeGpsRescueItemMinDthHelp"></div>
                                 </div>
+                                <div class="number">
+                                    <label> <input type="number" name="gps_rescue_initial_climb" min="5" max="1000" /> <span
```

Comment:
These are not the correct values for `gps_rescue_initial_climb`.  They should be min="0" max="100".
It's the `gps_rescue_return_altitude` that has the range 5 to 1000.